hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -190,10 +191,27 @@ public static void createCommitFile(String basePath, String commitTime) throws I
         } finally {
             os.close();
         }
+    }
 
+  public static void createSavepointFile(String basePath, String commitTime) throws IOException {
```

Comment:
Hmm Agreed. But there is a createCommitFile in that file and moving that as well to HoodieTestUtils touches a lot of tests. May be this can be a seperate PR.